packages/common/tests/params.ts

```diff
@@ -88,6 +88,28 @@ tape('[Common]: Parameter access', function (t: tape.Test) {
     st.end()
   })
 
+  t.test('EIP param access, paramByEIP()', function (st: tape.Test) {
+    const c = new Common('mainnet')
+    let f = function () {
+      c.paramByEIP('gasPrices', 'Bls12381G1AddGas', 'NOT_SUPPORTED_EIP')
+    }
+    let msg = 'Should throw for using paramByEIP() with a not supported EIP'
+    st.throws(f, /not supported$/, msg)
+
+    f = function () {
+      c.paramByEIP('notExistingTopic', 'Bls12381G1AddGas', 'EIP2537')
+    }
+    msg = 'Should throw for using paramByEIP() with a not existing topic'
+    st.throws(f, /not defined$/, msg)
+
+    f = function () {
+      c.paramByEIP('gasPrices', 'Bls12381G1AddGas_WITH_SPELLING_MISTAKE', 'EIP2537')
```

Comment:
Ah! Great that CodeCov reports that line 217 is not being ran - this function is not tested. Don't think I would have noticed otherwise that this function was not being called!

Reply:
@jochem-brouwer oh that is cool, this must be new! i haven't seen this "check warning" feature before.

<img width="1310" alt="Screen Shot 2020-09-03 at 8 47 02 AM" src="https://user-images.githubusercontent.com/22116/92137570-1e89b100-edc2-11ea-8112-5a1213da664b.png">

Reply:
@jochem-brouwer great catch, updated! 😄  (I am pretty sure that I wrote at least the corresponding message at some point, not sure where this got lost again? 🤔  Anyhow.)